src/Storages/LiveView/StorageLiveView.cpp

```diff
@@ -385,7 +386,7 @@ bool StorageLiveView::getNewBlocks()
     auto builder = completeQuery(std::move(from));
     auto pipeline = QueryPipelineBuilder::getPipeline(std::move(builder));
 
-    PullingAsyncPipelineExecutor executor(pipeline);
+    PullingPipelineExecutor executor(pipeline);
```

Comment:
@KochetovNicolai, why would `PullingAsyncPipelineExecutor` handle blocks differently than `PullingPipelineExecutor`?

Reply:
It shouldn't. Probably, there is a missing check in `PullingPipelineExecutor`. (Which is unusual cause the exception is from Port).

We need to fix header anyway.

Reply:
Not sure where the header needs to be fixed. Any ideas?